tests/test_sensors.py

```diff
@@ -96,3 +96,17 @@ def test_sensors(scene, has_sem, sensor_type, sim, make_cfg_settings):
     assert np.linalg.norm(
         obs[sensor_type].astype(np.float) - gt.astype(np.float)
     ) < 1.5e-2 * np.linalg.norm(gt.astype(np.float)), f"Incorrect {sensor_type} output"
+
+
+def test_smoke_no_sensors(make_cfg_settings):
```

Comment:
Probably good to add a brief comment here to explain the interpretation of `test_smoke_no_sensors` (i.e. that the "no sensor" configuration should be a valid, non-crashing configuration), otherwise the test function naming is a little bit confusing.